lib/model/topic_list.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:flutter/foundation.dart';
+import '../api/route/channels.dart';
+import '../model/store.dart';
+
+class TopicListView extends ChangeNotifier {
+  TopicListView({required this.store, required this.streamId});
```

Comment:
This feels like a store since it is where the data actually lives on the client, whereas `MessageListView`'s data comes from `MessageStore`.